config/database.codeship.yml

```diff
@@ -1,7 +1,7 @@
 development:
   adapter: mysql2
   host: 127.0.0.1
-  encoding: utf8
+  encoding: utf8mb4
```

Comment:
I would add  collation: utf8mb4_unicode_ci as well to these bits as we have that on staging